Source/Core/Core/HW/GCMemcard.h

```diff
@@ -116,7 +116,7 @@ struct Header           //Offset    Size    Description
 	// end Serial in libogc
 	u8 deviceID[2];     //0x0020    2       0 if formated in slot A 1 if formated in slot B
 	u8 SizeMb[2];       //0x0022    2       Size of memcard in Mbits
-	u16 Encoding;       //0x0024    2       Encoding (ASCII or japanese)
+	u16 Encoding;       //0x0024    2       Encoding (ASCII or Japanese)
```

Comment:
"Japanese" isn't an encoding anyway. I think this should say `Windows-1252 or Shift-JIS` instead.